app/views/history/cms/backups/_menu.html.erb

```diff
@@ -1,4 +1,4 @@
 <nav class="nav-menu">
-  <%= link_to :show, CGI.unescape(params[:source]) %>
-  <%= link_to :"history.restore", action: :restore if @item.data != @data %>
+  <%= link_to t('ss.links.show'), CGI.unescape(params[:source]) %>
```

Comment:
__[brakeman]__
* Unsafe parameter value in link_to href [:ref](http://brakemanscanner.org/docs/warning_types/link_to_href)

<!-- Comment by [SideCI](https://sideci.com) -->